src/modules/aws_acm/entity/certificate.ts

```diff
@@ -13,64 +17,123 @@ export enum certificateStatusEnum {
   VALIDATION_TIMED_OUT = 'VALIDATION_TIMED_OUT',
 }
 
+/**
+ * @enum
+ * Specifies if the certificate has been generated by AWS or has been imported
+ */
+
 export enum certificateTypeEnum {
   AMAZON_ISSUED = 'AMAZON_ISSUED',
   IMPORTED = 'IMPORTED',
   // TODO: add private certs support
   // PRIVATE = "PRIVATE",
 }
 
+/**
+ * @enum
+ * Specifies if the certificate is available to be renewed or not
+ */
 export enum certificateRenewalEligibilityEnum {
   ELIGIBLE = 'ELIGIBLE',
   INELIGIBLE = 'INELIGIBLE',
 }
 
+/**
+ * Table to query for all AWS certificates in the system, managed by AWS ACM.
+ * Certificates can be read and deleted, but not created or modified. Instead certificates can be created
+ * by using RPCs.
```

Comment:
Should we refer to the concept of RPCs here? Maybe just say like: `... instead certificates can be created by using the following postgres functions <link here to the posgres fns docs>.`